DESCRIPTION

```diff
@@ -43,7 +42,9 @@ Suggests:
     stringr,
     testthat,
     covr,
-    vdiffr
+    vdiffr,
+    lifecycle
+Remotes: r-lib/xml2    
```

Comment:
Looks like this change might be required to pass the GH actions build. This might bring up some trouble in submitting to CRAN, though: https://cran.r-project.org/web/packages/devtools/vignettes/dependencies.html

Andrew received an email a few days ago about breaking changes to infer as a result of the new dplyr release being submitted to CRAN on May 1st. I was planning on making those fixes before the end of the weekend so we could get a new version up on CRAN before then.

Did you get the sense that the developmental version of `xml2` was one of the changes that fixed the build, @ismayc ?

Reply:
It was a piece that I'm pretty sure is needed (https://community.rstudio.com/t/github-actions-failing-on-mac-devel-because-xml2-is-not-available/61537). I haven't tried to remove it though to see if it works without it given the other changes I made.

Reply:
Gotcha! Will merge for now and work through this during CRAN prep after those dplyr fixes.